proto/viam/app/cloudslam/v1/cloud_slam.proto

```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
+
+package viam.app.cloudslam.v1;
+
+import "common/v1/common.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "go.viam.com/api/app/cloudslam/v1";
+
+service CloudSLAMService {
+  rpc StartMappingSession(StartMappingSessionRequest) returns (StartMappingSessionResponse);
+  rpc GetActiveMappingSessionsForRobot(GetActiveMappingSessionsForRobotRequest) returns (GetActiveMappingSessionsForRobotResponse);
+  rpc GetMappingSessionPointCloud(GetMappingSessionPointCloudRequest) returns (GetMappingSessionPointCloudResponse);
+  rpc ListMappingSessions(ListMappingSessionsRequest) returns (ListMappingSessionsResponse);
+  rpc StopMappingSession(StopMappingSessionRequest) returns (StopMappingSessionResponse);
+}
+
+// StartMappingSession
+
+message StartMappingSessionRequest {
+  // The config required to run slam. This includes a replay sensor and map package, and any fields specific to the slam algorithm.
+  google.protobuf.Struct slam_config = 1;
+  // Version to use for slam, defaults stable
+  string slam_version = 2;
+  string map_name = 3;
+}
+
+message StartMappingSessionResponse {
+  string session_id = 1;
+}
+
+// GetActiveMappingSessionsForRobot
+
+message GetActiveMappingSessionsForRobotRequest {
+  // assumes only one active mapping session on a robot
+  string robot_id = 1;
+}
+
+message GetActiveMappingSessionsForRobotResponse {
+  string session_id = 1;
+}
+
+// GetMappingSessionPointCloud
+
+message GetMappingSessionPointCloudRequest {
+  string session_id = 1;
+}
+
+message GetMappingSessionPointCloudResponse {
+  // url to the pointcloud map
+  string url = 1;
+  // Current position within the SLAM Map
+  common.v1.Pose pose = 2;
+}
+
+// ListMappingSessions
+
+message ListMappingSessionsRequest {
+  string org_id = 1;
+  string location_id = 2;
+}
+
+message ListMappingSessionsResponse {
+  repeated MappingMetadata session = 1;
+}
+
+// StopMappingSession
+
+message StopMappingSessionRequest {
+  string session_id = 1;
+  bool save_map = 2;
+}
+
+message StopMappingSessionResponse {
+  string package_id = 1;
+  string version = 2;
```

Comment:
[q] What version is this? The cloud slam version? Slam version? Rdk version? 

If it is the cloud slam version, I think we should store that in the package

Reply:
I don't remember which version this was either - I think it must have been the package version. We should confirm with @JohnN193 once he's back. I'm ok to leave it in this PR, but added a comment on the scope.

Reply:
also unsure, will leave in